bundle_util.ts

```diff
@@ -36,3 +61,33 @@ export function setImportMap(importMap: string) {
 export function getImportMap() {
   return _importMap;
 }
+
+let _tsconfig: string | undefined;
+
+export function setTsconfig(tsconfig: string) {
+  _tsconfig = tsconfig;
+}
+
+export async function getTsconfig() {
+  if (!_tsconfig) {
+    switch (true) {
```

Comment:
This idiom looks confusing. Can you rewrite this to `if-else` blocks

Reply:
Done!